sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/AppendBlobAsyncApiTests.java

```diff
@@ -188,25 +191,17 @@ public void createAC(OffsetDateTime modified, OffsetDateTime unmodified, String
         t.put("foo", "bar");
 
         Mono<Response<AppendBlobItem>> response = bc.setTags(t)
-            .then(Mono.zip(setupBlobLeaseCondition(bc, leaseID), setupBlobMatchCondition(bc, match)))
-            .flatMap(tuple -> {
-                String newLease = tuple.getT1();
-                String newMatch = tuple.getT2();
-                if ("null".equals(newLease)) {
-                    newLease = null;
-                }
-                if ("null".equals(newMatch)) {
-                    newMatch = null;
-                }
-                BlobRequestConditions bac = new BlobRequestConditions()
-                    .setLeaseId(newLease)
-                    .setIfMatch(newMatch)
-                    .setIfNoneMatch(noneMatch)
-                    .setIfModifiedSince(modified)
-                    .setIfUnmodifiedSince(unmodified)
-                    .setTagsConditions(tags);
-                return bc.createWithResponse(null, null, bac);
-            });
+            .then(Mono.zip(setupBlobLeaseCondition(bc, leaseID), setupBlobMatchCondition(bc, match), BlobTestBase::convertNulls))
+                .flatMap(list -> {
```

Comment:
Instead of using `list` for the parameter name in these tests, could we name it something more self-documenting like `conditions`? I think that will help in being able to follow what's being passed around in these tests better.